HMCL/src/main/resources/assets/lang/I18N.properties

```
@@ -1144,6 +1144,30 @@ repositories.chooser.title=Choose download source for JavaFX
 resourcepack=Resource Packs
 resourcepack.download.title=Download Resource Pack - %1s
 
+reveal.in_explorer=Reveal in explorer
```

*Non-Windows platforms will most likely not use “Explorer”.*

```suggestion
reveal.in_explorer=Reveal in File Manager
```